packages/universal-components/src/Button/styles/shared.js

```diff
@@ -19,3 +19,54 @@ export const wrapperColor = {
   facebook: defaultTokens.backgroundButtonFacebook,
   google: defaultTokens.backgroundButtonGoogle,
 };
+
+export const size = {
+  small: {
+    height: parseInt(defaultTokens.heightButtonSmall, 10),
+    fontSize: parseInt(defaultTokens.fontSizeButtonSmall, 10),
+    paddingStart: {
+      paddingButton: parseInt(defaultTokens.spaceSmall, 10),
+      paddingButtonWithIcons: parseInt(defaultTokens.spaceXSmall, 10),
+      paddingButtonWithRightIcon: parseInt(defaultTokens.spaceSmall, 10),
+      paddingButtonWithLeftIcon: parseInt(defaultTokens.spaceMedium, 10),
```

Comment:
Is this value correct? It looks bigger than it should be.

<img width="152" alt="Screenshot 2019-04-02 at 11 12 48" src="https://user-images.githubusercontent.com/2660330/55391002-b97cbc80-5538-11e9-951f-c3daa0329b73.png">

Reply:
It should be, I was taking them from design-tokens and referencing orbit button values
Which value exactly it is?

Reply:
Was it as small button?

Reply:
Yes, it is small button. It just looked suspicious to me that there are just two size variants for other sizes and small is mixing three values. 
Left padding in Orbit storybook looks smaller for similar case.
<img width="93" alt="Screenshot 2019-04-02 at 12 34 31" src="https://user-images.githubusercontent.com/2660330/55396242-b935ee80-5543-11e9-8a43-e08de4a87325.png">

Reply:
Should be good now, after correcting values